configure

```diff
@@ -1118,8 +1120,8 @@ ARCH_SHARED_OBJS=''
 
 # Set ARCH specific FLAGS
 case "${ARCH}" in
-    # x86 specific optimizations
-    i386 | i486 | i586 | i686)
+    # x86/amd64 specific optimizations
+    i386 | i486 | i586 | i686 |x86_64)
```

Comment:
I assume with these changes now we always do SSE2 check for x64 instead of assuming it exists (like CMake) and that you have merged the x86 and x64 sections in configure?

Reply:
Yup.